### PR TITLE
NGINX 413 Payload Too Large Documentation

### PR DESCRIPTION
**Pull request by Myron-Woods-IBM**

### CODE REVIEW

File: docs/4-Transition/lessons.md [hunk 1]
```diff
@@ -47,13 +47,100 @@ Earlier versions of FileNet have the filesystem set to read only by default. How
...
+    - nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    - nginx.ingress.kubernetes.io/secure-backends: "true"
+    - nginx.ingress.kubernetes.io/session-cookie-name: route
+    - nginx.ingress.kubernetes.10/session-cookie-hash: sha1
+    - kubernetes.io/ingress.class: nginx
+```
+
+When `nginx.ingress.kubernetes.io/proxy-body-size` is set to 0, this removes they payload limit size. With this change to the CR, we were able to create documents with content successfully. 
+
+#### **Summary**
```
Comment (faraz) [inline docs/4-Transition/lessons.md:144]: @Myron-Woods-IBM you dont need to bold `**summary** after establishing a headline in markdown using hashtags

**Review by faraz — CHANGES_REQUESTED** [review]
Fix markdown formatting to keep it consistent

**Comment by faraz** [comment]
Reviewed - Requires changes

File: docs/4-Transition/lessons.md (hunk 1 above)
Comment (Myron-Woods-IBM) [reply docs/4-Transition/lessons.md]: Updated and removed bold styling from headlines

**Review by faraz — APPROVED** [review]

**Comment by faraz** [comment]
@Myron-Woods-IBM rebase and re-submit

**Pull request merged**